nuntium/tests/writeitinstances_test.py

```diff
@@ -302,9 +302,8 @@ def test_message_creation_on_post_form(self):
         self.assertEquals(response.status_code, 200)
         new_messages = Message.objects.filter(subject='Fiera no está')
         self.assertTrue(new_messages.count() > 0)
-        self.assertEquals(len(response.context["form"].errors), 0)
 
-    def test_I_get_an_acknoledgement_for_creating_a_message(self):
+    def test_get_an_acknoledgement_for_creating_a_message(self):
```

Comment:
Might as well correct the spelling in the method name to 'acknowledgement' while you're changing it.